index.js

```diff
@@ -4,12 +4,25 @@ const ora = require("ora");
 const fetch = require("node-fetch");
 const termImg = require("term-img");
 const cheerio = require("cheerio");
+const { findPokedexNumber } = require("./lookup");
 
 (async () => {
   try {
     const args = process.argv.slice(2);
+    const pokemon1 = findPokedexNumber(args[0]);
+    const pokemon2 = findPokedexNumber(args[1]);
 
-    const url = ["https://pokemon.alexonsager.net", ...args].join("/");
+    if (!pokemon1) {
```

Comment:
https://pokemon.alexonsager.net returns Missingno for unknown Pokemon.

We can remove these two Errors to keep that functionality.

Reply:
Sure thing! I wasn't sure whether we wanted to keep this behaviour. I'll remove it :)